README.md

```diff
@@ -38,6 +38,15 @@ The Drush Launcher Phar is able to self update to the latest release.
 
 Mac OSX systems are welcome to use Homebrew to install - `brew install drush-launcher`. To upgrade, run `brew upgrade drush-launcher`.
 
+## Installation - Composer
+
+Drush Launcher can also be installed via Composer, which can be easier on some systems, especially where Composer binaries are already on the system `$PATH`.
+
+1. Run `composer global require drush/drush-launcher`.
+1. Make sure your Composer binary directory is in your path. The best way to do this will vary slightly based on OS:
+    1. On Mac, add the following line to `.bash_profile`: `export PATH="$PATH:$HOME/.composer/vendor/bin"`
+    1. On Ubuntu, add the following line to `.bashrc`: `PATH="$PATH:$HOME/.config/composer/vendor/bin"`
```

Comment:
I am not a ubuntu user but ``$HOME/.config/composer/`` looks wrong.

Reply:
@webflo Can you be more specific? I use the approach described in the README on a daily basis.

I suppose if we want to be precise (bordering on pedantic), you should modify your path in `.profile` rather than `.bashrc`, but this will fail in some edge cases if users have created a `.bash_profile` file (which obviates `.profile`)

Reply:
The Composer directory does float around depending on your system. I think that `$HOME/.config/composer` is correct for Ubuntu. Seems like @danepowell has tested this well also.

If you want platform-independent instructions, `composer config home` will print out the Composer home directory, whatever it is. Maybe something like:
```
export PATH="$PATH:$(composer config home)/vendor/bin"
```
Just thinking out loud; I have not tried the technique above. There'd be an extra exec on each new shell with this solution.